src/components/shared/nav/nav.jsx

```diff
@@ -1,5 +1,36 @@
+import Link from 'next/link';
+import EmailListArrow from '../svg/EmailListArrow/EmailListArrow';
+import NoraLogo from '../svg/NoraLogo/NoraLogo';
```

Comment:
lets create an alias for the svg folder 
https://nextjs.org/docs/advanced-features/module-path-aliases